lib/v2/njxzc/router.js

```diff
@@ -1,4 +1,6 @@
 module.exports = (router) => {
-    // 南京晓庄学院官网通知公告
+    // 南京晓庄学院通知公告
     router.get('/tzgg', require('./home'));
+    // 南京晓庄学院图书馆通知公告
+    router.get('/libtzgg', require('./lib'));
```

Comment:
```suggestion
    // 南京晓庄学院图书馆通知公告
    router.get('/libtzgg', require('./lib'));
    // 南京晓庄学院通知公告
    router.get('/tzgg', require('./home'));
```
Please insert new entries in alphabetical order